tests/exporters/test_exporters_tflite_cli.py

```diff
@@ -103,7 +106,7 @@ def test_all_models_tested(self):
         # if len(missing_models_set) > 0:
         #     self.fail(f"Not testing all models. Missing models: {missing_models_set}")
 
-    @parameterized.expand(_get_models_to_test(MODELS_TO_TEST_FOR_TFLITE))
+    @parameterized.expand(_get_models_to_test(PYTORCH_EXPORT_MODELS_TINY))
     @require_tf
     def test_exporters_cli_pytorch(self, test_name: str, model_name: str, task: str, shapes: str):
         self._tflite_export(test_name, model_name, task, shapes)
```

Comment:
Do you know why this one is 3-4x slower than the ONNX export? TFLite specific?

![image](https://user-images.githubusercontent.com/9808326/217366393-12b51770-2bbe-4305-bd93-bc660a31349f.png)

Reply:
@mht-sharma might have idea as he recently conducted some benchmarking tests with TFLite.

Reply:
I think this is linked with the fact that the export requires saving the TF model on disk and loading the TF SavedModel to perform the export.